arch/Mips/MipsInstPrinter.c

```diff
@@ -269,10 +278,12 @@ static void printMemOperand(MCInst *MI, int opNum, SStream *O)
 		break;
 	}
 
+	Mips_set_mem_access(MI, true);
 	printOperand((MCInst *)MI, opNum + 1, O);
 	SStream_concat0(O, "(");
 	printOperand((MCInst *)MI, opNum, O);
 	SStream_concat0(O, ")");
+	Mips_set_mem_access(MI, false);
```

Comment:
As discussed

Reply:
i have modified this following what also the other ops does.